internal/entitlement/static/static_entitlement.go

```diff
@@ -0,0 +1,26 @@
+package staticentitlement
```

Comment:
nit: I would name the file entitlement.go as it's obvious that it's the static one.

Reply:
i usually optimize names for my fuzzy finder but i suppose thats a unique concern to me (for that it finds it easier if it has this name)

Nevertheless i agree it would be nicer to call them all just entitlement.go, so i'll rename them